### PR TITLE
add `Do Not Sell Data` flag

### PR DESCRIPTION
**Pull request by vrabeshko**

### CODE REVIEW

**Comment by vrabeshko** [comment]
[L - Implement CCPA "Do Not Sell" flag](https://trello.com/c/G7RIObDq/1565-l-implement-ccpa-do-not-sell-flag)

File: API.md [hunk 1]
```diff
@@ -895,7 +896,23 @@ Either `campaignId` or `consentIds` should be provided.
 
 standard response object
 - status 200 if success:
-    - result will contains reference
+    - result will contain reference
+- status 400 if error
+
+## dontSellData(email[, config])
+Set `Do Not Sell` flag for user. Consents for this users won't be shared between campaigns for 1 year. After expiration it should be extended
```
Comment (dhurlburtusa) [inline API.md]:
```suggestion

Set `Do Not Sell` flag for user. Consents for this user won't be shared between campaigns for 1 year. After expiration, it should be extended.
```

File: src/XcooBee/Core/Api/Consents.php [hunk 2]
```diff
@@ -624,4 +624,37 @@ public function shareConsents($campaignRef, $campaignId = null, $consentIds = []
...
+     * @return Response
+     * @throws XcooBeeException
+     */
+    public function dontSellData($email, $config = [])
+    {
+        if (!$email) {
+            throw new XcooBeeException('No "email" provided');
+        }
+
+        $query = 'mutation dontSellData($email: String!){
```
Comment (dhurlburtusa) [inline src/XcooBee/Core/Api/Consents.php]:
```suggestion
        $query = 'mutation dontSellData($email: String!) {
```

File: src/XcooBee/Core/Api/Consents.php [hunk 3]
```diff
@@ -624,4 +624,37 @@ public function shareConsents($campaignRef, $campaignId = null, $consentIds = []
...
+            do_not_sell_data(email: $email){
```
Comment (dhurlburtusa) [inline src/XcooBee/Core/Api/Consents.php]:
```suggestion
            do_not_sell_data(email: $email) {
```

File: src/XcooBee/Core/Api/Consents.php [hunk 4]
```diff
@@ -624,4 +624,37 @@ public function shareConsents($campaignRef, $campaignId = null, $consentIds = []
...
+                user_email
```
Comment (dhurlburtusa) [inline src/XcooBee/Core/Api/Consents.php]:
```suggestion
              user_email
```

**Comment by vrabeshko** [comment]
@dhurlburtusa applied. Thanks!

**Pull request merged**